freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/instance/InstanceTemplateRequest.java

```diff
@@ -23,6 +24,10 @@
     @ApiModelProperty(FreeIpaModelDescriptions.AWS_PARAMETERS)
     private AwsInstanceTemplateParameters aws;
 
+    @Valid
+    @ApiModelProperty(FreeIpaModelDescriptions.AZURE_PARAMETERS)
+    private AzureInstanceTemplateParameters azure;
+
```

Comment:
Like for `AzureInstanceTemplateParameters` and `AzureInstanceTemplateEncryptionParameters`, I would remove this property and instead extract encryption details from the `EnvironmentResponse`.